sql/org_mozilla_firefox_derived/migrated_clients_v1/init.sql

```diff
@@ -3,40 +3,37 @@ CREATE OR REPLACE TABLE
 PARTITION BY
   (submission_date)
 CLUSTER BY
-  (fenix_client_id)
-AS
-WITH migrations AS (
+  (fenix_client_id) AS
+WITH
+  migrations AS (
   SELECT
     client_info.client_id AS fenix_client_id,
     metrics.uuid.migration_telemetry_identifiers_fennec_client_id AS fennec_client_id,
     submission_date,
     normalized_channel,
+    client_info.device_manufacturer,
+    metadata.geo.country,
     versions.value AS value
   FROM
     `moz-fx-data-shared-prod.org_mozilla_firefox.migration`
   LEFT JOIN
     UNNEST(metrics.labeled_string.migration_migration_versions) versions
   WHERE
-    submission_date
-    BETWEEN '2020-01-01'
-    AND current_date
-),
-clients AS (
+    submission_date BETWEEN '2020-01-01'
+    AND current_date ),
+  clients AS (
   SELECT
     fenix_client_id,
     MAX(submission_date) AS submission_date,
-    `moz-fx-data-shared-prod.udf.mode_last`(
-      ARRAY_AGG(normalized_channel IGNORE NULLS)
-    ) AS normalized_channel,
-    `moz-fx-data-shared-prod.udf.mode_last`(
-      ARRAY_AGG(fennec_client_id IGNORE NULLS)
-    ) AS fennec_client_id,
+    `moz-fx-data-shared-prod.udf.mode_last`( ARRAY_AGG(normalized_channel IGNORE NULLS) ) AS normalized_channel,
```

Comment:
It would be preferred to not use the project prefix here. That's only needed in view definitions.